requirements.txt

```diff
@@ -1,21 +1,23 @@
-boto==2.31.1
-konfig==0.9
-Paste==1.7.5.1
+boto>=2.31.1
+ipdb
```

Comment:
I'm glad you mentioned about this. I agree that it should not be necessary for prod deployments but, since we don't have `requirements-dev.txt`, I went ahead and removed the ipdb requirement on the `requirements.txt`. Let me know if you think it will be better to create a `requirements-dev.txt` file.